src/RoslynAnalyzers/Tools/GenerateDocumentationAndConfigFiles/Program.cs

```diff
@@ -615,12 +617,13 @@ Rule ID | Missing Help Link | Title |
                     DiagnosticDescriptor descriptor = ruleById.Value;
 
                     var helpLinkUri = descriptor.HelpLinkUri;
-                    if (!string.IsNullOrWhiteSpace(helpLinkUri) &&
-                        await checkHelpLinkAsync(helpLinkUri).ConfigureAwait(false))
-                    {
-                        // Rule with valid documentation link
+                    if (string.IsNullOrWhiteSpace(helpLinkUri))
+                        continue;
```

Comment:
unless I am misreading this, we will now pass validation if the help link is missing, which isn't what we did before.  seems like a missing link should fail validation

Reply:
Yep, you're right. Will fix.